composer.json

```diff
@@ -4,21 +4,22 @@
     "description": "SDK for AsyncAPI.",
     "license": "proprietary",
     "require": {
-        "php": ">=7.4",
-        "spryker-sdk/spryk": "^0.4.0",
+        "php": ">=8.0",
+        "doctrine/inflector": "^2.0",
```

Comment:
This breaks master CI for prefer lowest
https://app.travis-ci.com/github/spryker/suite-nonsplit/jobs/595562351

The core/project expects also v1 to work
Please fix this
@stereomon and team

Reply:
`async-api` package is not really required for suite-nonsplit so could be just removed. @stereomon

Reply:
Please do so then with emergency PR merge only asap

Also, this happened because of https://release.spryker.com/release-groups/view/4612 missing project run
Avoid this in the future please!

Reply:
But I disagree with the assessment
You would force on split side at least the v2 upgrade even if some other modules are still compatible to both and must remain this to allow the max compatibility range that we are expected to keep.
So instead this should be fixed here IMO.

Reply:
https://github.com/spryker/suite-nonsplit/commit/1de9b34c4ac5b939afe03b4109f30bdcdc83bbf4 done

Reply:
I agree that we can remove it from nonsplit in a follow up PR.

Reply:
Ah now I see why it cannot be removed from project and must be fixed in this repo:

    "require": {
        "doctrine/inflector": "~1.4.2 || ~2.0.0",
        "php": ">=8.0"
    },
    
spryker/doctrine-inflector Module, which is used by 4+ other modules ( https://release.spryker.com/modules/dependencies/529 ) requires 1 || 2
So must your vendor code right now.

@matweew
Please schedule the follow up fix for this and re-enable in project level as per this validation check
https://app.travis-ci.com/github/spryker/suite-nonsplit/jobs/595587499#L3469

Reply:
https://release.spryker.com/release-groups/view/4664